### PR TITLE
add fancy changelog dialog

### PR DESCRIPTION
**Pull request by aviraldg**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -58,7 +58,8 @@
     "react-gemini-scrollbar": "matrix-org/react-gemini-scrollbar#5e97aef",
     "sanitize-html": "^1.11.1",
     "ua-parser-js": "^0.7.10",
-    "url": "^0.11.0"
+    "url": "^0.11.0",
+    "whatwg-fetch": "^1.0.0"
```
Comment (ara4n) [inline package.json]: hang on, why do we need `whatwg-fetch` when we already have `browser-request`?

File: src/components/views/globals/NewVersionBar.js [hunk 2]
```diff
@@ -17,20 +17,36 @@ limitations under the License.
...
+                    window.location.reload();
+                }
+            }
+        });
+    };
+
+    return (
+        <div className="mx_MatrixToolbar">
+            <img className="mx_MatrixToolbar_warning" src="img/warning.svg" width="24" height="23" alt="/!\"/>
+            <div style={{flex: 1}}>
```
Comment (ara4n) [inline src/components/views/globals/NewVersionBar.js]: ew ew ew. please let's not do inlined CSS. i thought @richvdh already fixed the CSS for this?

**Comment by ara4n** [comment]
lgtm other than comments. i'm not *entirely* sure it was worth spidering github for this info; i'd have just linked through to the appropriate changelog pages, but happy to keep it now it exists ;)

**Comment by ara4n** [comment]
i'd really rather that we didn't sprout two HTTP clients in vector-web/matrix-react-sdk; please either convert browser-request to whatwg-fetch or vice versa, before the proliferation gets worse.

**Comment by ara4n** [comment]
@aviraldg please can you get rid of the inline styles? and then this is good to go.

**Comment by ara4n** [comment]
lgtm

**Comment by richvdh** [comment]
Given react-sdk uses whatwg-fetch, courtesy of https://github.com/matrix-org/matrix-react-sdk/pull/296, as do some of its dependencies, I think the horse has already bolted on having two HTTP clients. But still, I'd rather see browser-request than whatwg here.

**Pull request merged**